vertical-pod-autoscaler/pkg/recommender/input/controller_fetcher/controller_fetcher.go

```diff
@@ -267,6 +268,9 @@ func (f *controllerFetcher) isWellKnownOrScalable(key *ControllerKeyWithAPIVersi
 
 	for _, mapping := range mappings {
 		groupResource := mapping.Resource.GroupResource()
+		if gk, err := key.groupKind(); err != nil && gk.Kind == node {
```

Comment:
Why here and not above (around line 256, where the comment "if not well known" is)?
It seems that key does not change between there & here.

Reply:
Done